manifests/postgresql.crd.yaml

```diff
@@ -469,6 +469,8 @@ spec:
                     type: string
                   standby_port:
                     type: string
+                  standby_primary_slot_name:
+                    type: string
```

Comment:
please also copy it to the postgres CRD of the [helm chart](https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/crds/postgresqls.yaml)

Reply:
Already done: https://github.com/zalando/postgres-operator/blob/e209650f2f2ca1bca1236116a8896e07ee48d777/charts/postgres-operator/crds/postgresqls.yaml#L474